### PR TITLE
Update privacy notice page to use new table

### PR DESCRIPTION
**Pull request by Kelsey-Ethyca**
Closes #[PROD-1455](https://ethyca.atlassian.net/browse/PROD-1455)

### Description Of Changes

Update privacy notice page to use new table component

Before: 
fides-plus-rc fides-staging ethyca com_consent_privacy-notices (1)
After:
localhost_3000_consent_privacy-notices (2)

### Code Changes

* [ ] update `PrivacyNoticesTable` to use the new v2 table 
* [ ] updates to privacy-notice/cells and v2/cells so the new notice table is working with the proper cells 
* [ ] update cypress tests to pass 

### Steps to Confirm

* [ ] run plus branch `consent_multitranslation_plus` from https://github.com/ethyca/fidesplus/pull/1299 with `nox -s "build(slim)" "dev(slim)" -- dev`
* [ ] run this branch with `cd client && turbo run dev`
* [ ] nav to the notices page 
* [ ] add systems with data uses 
* [ ] toggle notices enabled/disabled 
* [ ] click into notices by clicking on the row 
* [ ] ensure the columns are correct - Title, Mechanism, Location, Status, Enable
* [ ] ensure you can add and edit privacy notices 

The Last Update column, sorting, searching, and filtering will be handled next sprint in https://ethyca.atlassian.net/browse/PROD-1771

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`
* [ ] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

[PROD-1455]: https://ethyca.atlassian.net/browse/PROD-1455?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: clients/admin-ui/src/features/common/table/v2/cells.tsx [hunk 1]
```diff
@@ -6,10 +6,20 @@ import {
...
 } from "@fidesui/react";
 import { HeaderContext } from "@tanstack/react-table";
-import { FC, ReactNode } from "react";
+import { ChangeEvent, FC, ReactNode } from "react";
+
+import { RTKResult } from "~/types/errors";
+
+import { getErrorMessage, isErrorResult } from "../../helpers";
+import ConfirmationModal from "../../modals/ConfirmationModal";
+import { errorToastParams } from "../../toast";
```
Comment (galvana) [inline clients/admin-ui/src/features/common/table/v2/cells.tsx:22]: We prefer absolute paths relative to the base directory `~` so we should use
```
../../helpers -> ~/features/common/helpers
../../modals/ConfirmationModal -> ~/features/common/modals/ConfirmationModal
../../toast -> ~/features/common/toast
```
This way the imports will always be accurate even if the components are refactored to be in a different location relative to each other

File: clients/admin-ui/src/features/privacy-notices/PrivacyNoticesTable.tsx [hunk 2]
```diff
@@ -1,110 +1,247 @@
...
+    items: data,
+    total: totalRows,
+    pages: totalPages,
+  } = useMemo(() => notices || emptyNoticeResponse, [notices]);
+
+  useEffect(() => {
+    setTotalPages(totalPages);
+  }, [totalPages, setTotalPages]);
+
+  const inventoryColumns: ColumnDef<LimitedPrivacyNoticeResponseSchema, any>[] =
```
Comment (galvana) [inline clients/admin-ui/src/features/privacy-notices/PrivacyNoticesTable.tsx:126]:
```suggestion
  const privacyNoticeColumns: ColumnDef<LimitedPrivacyNoticeResponseSchema, any>[] =
```

**Review by galvana — CHANGES_REQUESTED** [review]
This looks good but just make sure to update the import paths to be absolute `~` wherever you added new imports. I usually reference other places where those components are being imported to verify the paths.

File: clients/admin-ui/src/features/privacy-notices/PrivacyNoticesTable.tsx (hunk 2 above)
Comment (Kelsey-Ethyca) [reply clients/admin-ui/src/features/privacy-notices/PrivacyNoticesTable.tsx]: ✅

**Comment by galvana** [comment]
@Kelsey-Ethyca don't forget the change log!

**Pull request merged**